fileprovider/src/main/res/values-zh/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="file_shuttle_summary">炼妖壶·彼界</string>
```

Comment:
why change this file? this seems to be dialect-unrelated zh translation?

Reply:
Sorry.That's my mistake.
I will restore it back.